lang/python/python/files/python-package-setuptools.mk

```diff
@@ -12,7 +12,7 @@ $(call Package/python/Default)
   LICENSE:=MIT
   LICENSE_FILES:=LICENSE
 #  CPE_ID:=cpe:/a:python:setuptools # not currently handled this way by uscan
-  DEPENDS:=+python
+  DEPENDS:=+python +python-pkg-resources
 endef
 
 define PyPackage/python-setuptools/install
```

Comment:
I think there is also a line in `define PyPackage/python-setuptools/install` that copies `pkg_resources`?

Reply:
oops; thanks for catching this;